x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -45,18 +46,10 @@ const PaddedContainer = styled.div`
   padding: ${px(units.plus)} ${px(units.plus)} 0;
 `;
 
-const Container = styled.div`
-  position: relative;
-  border: 1px solid ${theme.euiColorLightShade};
-  border-radius: ${borderRadius};
-  margin-top: ${px(units.plus)};
-`;
```

Comment:
Feels good seeing this custom container go away. I hope we can do this everywhere. Same goes for custom headers.

Reply:
Yeah, we should be able to remove in other places too.